net-dns/dnsviz/dnsviz-0.8.2.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python{2_7,3_4,3_5,3_6} )
+inherit distutils-r1 eutils
+
+DESCRIPTION="Tool suite for analysis and visualization of DNS and DNSSEC"
+HOMEPAGE="http://dnsviz.net/"
+SRC_URI="https://github.com/dnsviz/dnsviz/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+SLOT="0"
+LICENSE="GPL-2"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+DEPEND="
+	dev-python/dnspython[${PYTHON_USEDEP}]
+	dev-python/libnacl[${PYTHON_USEDEP}]
+	>=dev-python/m2crypto-0.31.0[${PYTHON_USEDEP}]
+	>=dev-python/pygraphviz-1.3.1[${PYTHON_USEDEP}]
+	dev-python/setuptools[${PYTHON_USEDEP}]"
+
+RDEPEND="
+	${DEPEND}"
+
+python_prepare_all() {
+	# Fix the ebuild to use correct FHS/Gentoo policy paths for 0.8.2
+	sed -i "s*share/doc/dnsviz*share/doc/dnsviz-${PV}*g" "${S}"/setup.py || die
+
+	distutils-r1_python_prepare_all
+}
+
+pkg_postinst() {
+	elog "Support for extra feature can be get from:"
+	optfeature "Support for pre-deployment testing" net-dns/bind
+
+	# Get the current OpenSSL VERSION for GOST Engine requirements
+	openssl_version_major="$(openssl version | cut -d " " -f 2 | cut -c 1)"
+	openssl_version_minor="$(openssl version | cut -d " " -f 2 | cut -c 3)"
+
+	# Warn about extra requirements for >=OpenSSL 1.1.0
+	if [[ ${openssl_version_major} -ge 1 ]] &&
```

Comment:
I am more than sure that the whole block can be replaced with the `if has_version`

Reply:
@Zlogene does` has_version` allow to wildcard or just return 1 for exact match I couldn't find the source code of this function.

`if has_version >=dev-libs/openssl-1.1.*`

Because all openssl packages have extra string at the end.
1.1.1b-r2	
1.1.0j-r1	
1.0.2r-r200	
1.0.2r	
0.9.8z_p8-r1